src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs

```diff
@@ -17,7 +17,7 @@
 using Microsoft.Azure.Management.Storage;
 using System;
 
-namespace Microsoft.Azure.Commands.Compute
+namespace Microsoft.Azure.Commands.Management.Storage
```

Comment:
@huangpf , can you reply in the mail thread I just forward to you? We can discuss and make a decision in the mail thread.  
BTW, maybe next time you can raise your concern earlier before I sent the PR...:-)